test/import/importConfiguration.test.ts

```diff
@@ -365,23 +367,38 @@ describe('importConfiguration', () => {
     });
   });
 
-  describe('#url', () => {
-    it('should import url as-is', () => {
-      minYAML.url = 'http://foo.org/some-url';
+  describe('#canonical', () => {
+    it('should import canonical as-is', () => {
+      minYAML.canonical = 'http://foo.org/some-canonical-url';
       const config = importConfiguration(minYAML, 'test-config.yaml');
-      expect(config.url).toBe('http://foo.org/some-url');
+      expect(config.canonical).toBe('http://foo.org/some-canonical-url');
     });
-    it('should report an error and throw if url is missing', () => {
-      delete minYAML.url;
+    it('should report an error and throw if canonical is missing', () => {
+      delete minYAML.canonical;
       expect(() => importConfiguration(minYAML, 'test-config.yaml')).toThrow(
         'Minimal config not met'
       );
       expect(loggerSpy.getLastMessage('error')).toMatch(
-        /SUSHI minimally requires the following configuration properties to start processing FSH: id, version, url, fhirVersion\.\s*File: test-config\.yaml/
+        /SUSHI minimally requires the following configuration properties to start processing FSH: id, version, canonical, fhirVersion\.\s*File: test-config\.yaml/
       );
     });
   });
 
+  describe('#url', () => {
+    it('should import url as-is if provided', () => {
+      minYAML.url = 'http://foo.org/some-url/ImplementationGuide/my.guide';
+      const config = importConfiguration(minYAML, 'test-config.yaml');
+      expect(config.url).toBe('http://foo.org/some-url/ImplementationGuide/my.guide');
+    });
+    it('should default url based on canonical if url is not provided', () => {
+      // loggerSpy.reset(); // JULIA
```

Comment:
JULIA

Reply:
OMG NOOOO

Reply:
I will be force pushing to remove all evidence of this.